src/components/styles/Company.css

```diff
@@ -31,16 +30,22 @@ body {
 }
 
 .company-card:hover {
-  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
   transform: translateY(-2px);
-  border: 2px solid #fff;
 }
 
 a {
   text-decoration: none;
   color: #646970;
 }
 
+/* .company-card:nth-child(8n + 2),
+.company-card:nth-child(8n + 4),
+.company-card:nth-child(8n + 5),
+.company-card:nth-child(8n + 7) {
+  background-color: rgb(197, 217, 237);
+  border: 2px solid #fff;
+} */
+
```

Comment:
- Hey, great job addressing all the issues pointed out by the previous reviewer, however, you've added these commented pieces of code that shouldn't be in your code base as they make your codebase not clean and maintainable. Please remove them, as this is the only reason your project won't be approved.

Reply:
I've made changes. Thank you